_docker/awestruct/Dockerfile

```diff
@@ -6,6 +6,15 @@ ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US.UTF-8
 ENV LC_ALL en_US.UTF-8
 
+RUN yum install -y  \
+  pngquant
+
+RUN wget http://prdownloads.sourceforge.net/optipng/optipng-0.7.5.tar.gz?download
+RUN mv optipng* optipng.tar.gz
+RUN tar -xvzf optipng.tar.gz
+WORKDIR /tmp/optipng-0.7.5
+RUN ./configure && make && make install
```

Comment:
Any reason not to combine more of the optipng downloading and building into a single step?